src/en/installconfig-checklist.md

```diff
@@ -85,23 +85,20 @@ communicate properly with the region API server. [Commission][commission-nodes]
 your node now.
 
 
-## Deploy a node directly
+## Deploy a node
 
 Lots of folks would have [Juju][juju-getting-started] take over at this point.
```

Comment:
Perhaps https://jujucharms.com/docs/ and let that determine the ultimate destination??